src/java/Military.java

```diff
@@ -0,0 +1,34 @@
+package java;
+
+public class Military extends Air_transport {
+    boolean EjectionSystems;
+    int NumberOfMissiles;
+
+    @Override
+    void Vyvod() {
+        super.Vyvod();
+        if (EjectionSystems){
+        System.out.println("Ejection Systems: yes ");
+        }
+        System.out.println("Number of Missiles: " + NumberOfMissiles);
+    }
+    void shot() {
+        if (NumberOfMissiles != 0) {
+            System.out.println("Missile fired!");}
+        else{
+            System.out.println("No ammunition!");
+        }
+    }
+    void ejection(){
+        if (EjectionSystems){
+            System.out.println("Bailout Successful");}
```

Comment:
@Hlieb Чутка не красиво так
```suggestion
            System.out.println("Bailout Successful");
      }
```